src/no-random-v4-uuid.spec.ts

```diff
@@ -0,0 +1,94 @@
+// no-random-v4-uuid.spec.ts
+
+import { RuleTester } from '@typescript-eslint/rule-tester';
+import { describe } from '@jest/globals';
+import rule, { ruleId } from './no-random-v4-uuid';
+
+describe(ruleId, () => {
```

Comment:
describe is no longer necessary because rule-tester manages these test cases automatically